cmd/notary/tuf.go

```diff
@@ -133,6 +136,7 @@ type tufCommander struct {
 func (t *tufCommander) AddToCommand(cmd *cobra.Command) {
 	cmdTUFInit := cmdTUFInitTemplate.ToCommand(t.tufInit)
 	cmdTUFInit.Flags().StringVar(&t.rootKey, "rootkey", "", "Root key to initialize the repository with")
+	cmdTUFInit.Flags().StringVar(&t.rootCert, "rootcert", "", "Root certificate to initialize the repository with")
```

Comment:
It looks like a root cert could be provided without a root key, so long as that root cert corresponds to a private key that is in the CryptoStore.

But if both a key and a cert are provided, they must match (e.g. the cert must correspond to the key provided). Agree with @riyazdf though that this would be good to include in the comment, since it's possible someone could interpret this as "add this new root key, and also use this cert which corresponds to a key already in the key store".